directory/directory.js

```diff
@@ -452,6 +451,10 @@ directory.addPair({
   baseBuying: ['XLM', 'native'],
   counterSelling: ['BTC', 'golix.io'],
 });
+directory.addPair({
+  baseBuying: ['XLM', 'native'],
+  counterSelling: ['BTC', 'charnatoken.top'],
```

Comment:
CHRC is not a BTC. It should be under "Pairs involving XLM"

Reply:
okay . but we want to exchange CHRC with btc is that possible?